lib/elixir_boilerplate_web/graphql_router.ex

```diff
@@ -0,0 +1,14 @@
+defmodule ElixirBoilerplateWeb.GraphQLRouter do
+  use Plug.Router
+
+  @absinthe_configuration [document_providers: {ElixirBoilerplateGraphQL, :document_providers}, json_codec: Jason, schema: ElixirBoilerplateGraphQL.Schema]
+
+  plug(ElixirBoilerplateGraphQL.Plugs.Context)
+  plug(:match)
+  plug(:dispatch)
```

Comment:
Where is the plug definition located?

Reply:
This is from [`Plug.Router`](https://hexdocs.pm/plug/Plug.Router.html).

Reply:
Of course 😳